tutorials/plot_eeg_no_mri.py

```diff
@@ -0,0 +1,126 @@
+
+# Read and set the EEG electrode locations
+montage = mne.channels.read_montage('standard_1005', ch_names=raw.ch_names,
+                                    transform=True)
+raw.set_montage(montage)
+
+
+subject = 'fsaverage'
+
+# Check that the locations of EEG electrodes is correct with respect to MRI
+mne.viz.plot_alignment(raw.info, subject=subject, eeg="projected",
+                       subjects_dir=subjects_dir, trans=trans_fname)
+
+##############################################################################
+# Setup source space and compute forward
+# --------------------------------------
+
+src = mne.setup_source_space(subject, spacing='oct6',
+                             subjects_dir=subjects_dir, add_dist=False)
+print(src)
+
+mne.viz.plot_bem(subject=subject, subjects_dir=subjects_dir,
+                 brain_surfaces='white', src=src, orientation='coronal')
+
+conductivity = (0.3, 0.006, 0.3)  # for three layers
+model = mne.make_bem_model(subject=subject, ico=4,
+                           conductivity=conductivity,
+                           subjects_dir=subjects_dir)
+bem = mne.make_bem_solution(model)
+
+fwd = mne.make_forward_solution(raw.info, trans=trans_fname, src=src,
+                                bem=bem, eeg=True, mindist=5.0, n_jobs=2)
+print(fwd)
+
+##############################################################################
+# Epoch data
+# ----------
+
+tmin, tmax = -1, 4  # define epochs around events (in s)
+event_ids = dict(hands=2, feet=3)  # map event IDs to tasks
+events, _ = mne.events_from_annotations(raw)
+raw.filter(1., 40.)
+raw.set_eeg_reference(projection=True)
+epochs = mne.Epochs(raw, events, event_ids, tmin - 0.5, tmax + 0.5,
+                    picks=('eeg'), baseline=None, preload=True)
+
+evoked = epochs.average()
+
+##############################################################################
+# Compute dSPM source estimates
+# -----------------------------
+
+# Compute noise cov
+noise_cov = mne.compute_covariance(
+    epochs, tmax=0., method=['shrunk', 'empirical'], rank='full')
+
+# make an MEG inverse operator
+info = evoked.info
+inverse_operator = make_inverse_operator(info, fwd, noise_cov,
+                                         loose=0.2, depth=0.8)
+
+
+method = "dSPM"
+snr = 3.
+lambda2 = 1. / snr ** 2
+stc, residual = apply_inverse(evoked, inverse_operator, lambda2,
+                              method=method, pick_ori=None,
+                              return_residual=True, verbose=True)
+
+###############################################################################
+# Now plot the source estimates
+# -----------------------------
+
+vertno_max, time_max = stc.get_peak(hemi='rh')
+
+subjects_dir = data_path + '/subjects'
+surfer_kwargs = dict(
+    hemi='rh', subjects_dir=subjects_dir, views='lateral',
+    initial_time=time_max, time_unit='s', size=(800, 800), smoothing_steps=5)
+brain = stc.plot(**surfer_kwargs)
```

Comment:
why separate into `surfer_kwargs dict`? Would be simpler to have these directly in the function call.